.gitmodules

```diff
@@ -360,3 +360,9 @@
 [submodule "contrib/sqids-cpp"]
 	path = contrib/sqids-cpp
 	url = https://github.com/sqids/sqids-cpp.git
+[submodule "contrib/QAT-ZSTD-Plugin"]
```

Comment:
Could you please order the new submodules below the existing `qpl`/`idxd-config` submodules to keep things together.

Reply:
Just wondering, why is QAT-ZSTD not shipped directly with qatlib? Any technical or legal reasons for that?

Reply:
The purpose is to be compatible with two kinds of setup: out-of-tree driver and in-tree library(qatlib). Out-of-tree driver means kernel don't have native support, user will download package [here](https://www.intel.com/content/www/us/en/download/765501/intel-quickassist-technology-driver-for-linux-hw-version-2-0.html), install qat driver from source.
Please look into details [here](https://www.intel.com/content/www/us/en/download/765501/intel-quickassist-technology-driver-for-linux-hw-version-2-0.html)

Reply:
> Could you please order the new submodules below the existing `qpl`/`idxd-config` submodules to keep things together.

done